roles/download/defaults/main.yml

```diff
@@ -8,6 +8,8 @@ calico_version: v0.19.0
 calico_cni_version: v1.2.1
 weave_version: v1.5.0
 
+cni_version: "v0.2.2"
```

Comment:
Could you please change the variable name to something like `fan_cni_version`?
Please refer to calico and weave.